src/App.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import AddDoctor from './pages/AddDoctor';
+import Doctors from './pages/Doctors';
+import AddReservation from './pages/AddReservation';
+import DeleteDoctor from './pages/DeleteDoctor';
+import DoctorDetails from './pages/DoctorDetails';
+import MyReservations from './pages/MyReservations';
+import NavPanel from './components/NavPanel';
+import LogIn from './pages/LogIn';
+import './App.css';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```
```javascript
const renderApp= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.

![image](https://user-images.githubusercontent.com/57421823/223504481-60a73ac6-cf89-4921-8eda-e271d03fa375.png)